plugins/hls-tactics-plugin/src/Ide/Plugin/Tactic/TestTypes.hs

```diff
@@ -33,20 +34,23 @@ tacticTitle UseDataCon dcon          = "Use constructor " <> dcon
 
 ------------------------------------------------------------------------------
 -- | Plugin configuration for tactics
-newtype Config = Config
-  { cfg_feature_set :: FeatureSet
+data Config = Config
+  { cfg_feature_set          :: FeatureSet
+  , cfg_max_use_ctor_actions :: Int
```

Comment:
You might want to make the new field optional to allow for partial configs

Reply:
Good catch. But I'd rather use a loose parser than juggle maybes everywhere. Will implement.